static/app/components/replays/replayContext.tsx

```diff
@@ -299,4 +299,4 @@ export function Provider({children, events, value = {}}: React.PropsWithChildren
   );
 }
 
-export const Consumer = ReplayPlayerContext.Consumer;
+export const useReplayContext = () => useContext(ReplayPlayerContext);
```

Comment:
Make a simple export instead of exporting `ReplayPlayerContext`. 

Also, we don't want to export `ReplayPlayerContext.Provider` because then all the provider logic would be skipped!

In test/stories we can still override the prodider logic because `value` is a prop that's passed through into the internal call to `<ReplayPlayerContext.Provider>`